src/convertSKU.ts

```diff
@@ -164,6 +181,8 @@ const convertSKU = (product: BiggySearchProduct, indexingType?: IndexingType, tr
 
   const variations = getVariations(sku, product)
 
+  const attributes = convertAttributes(sku.catalogAttributes ?? [])
```

Comment:
using the same type you can simply do:
```suggestion
  const attributes = sku.catalogAttributes ?? []
```